services/audit-service/src/main/java/datawave/microservice/audit/auditors/file/config/FileAuditProperties.java

```diff
@@ -0,0 +1,83 @@
+package datawave.microservice.audit.auditors.file.config;
+
+import datawave.microservice.audit.config.AuditProperties;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.Valid;
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.NotNull;
+
+@Validated
+public class FileAuditProperties {
+    
+    @NotNull
```

Comment:
`@NotBlank`, or is an empty string allowed for the pathUri?

Reply:
updated.